personal_assistant/contacts/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+class Contact(models.Model):
+    name = models.CharField(max_length=50, null=False)
+    address = models.CharField(max_length=150, null=True)
+    phone = models.CharField(max_length=12, null=False)
+    email = models.CharField(max_length=50, null=True)
+    birthday = models.DateField(null=True)
+
+    def __str__(self):
+        return f"{self.name}"
```

Comment:
Оскільки в рядку міститься лише одна поле класу то можна повертати безпосередньо його:
```suggestion
        return self.name
```